packages/types/src/availabilitySlot.ts

```diff
@@ -40,3 +46,38 @@ export type SubSlot = {
 };
 
 export type GeneralSlot = Slot | SubSlot;
+
+// API Payloads / Queries
+export type FindAvailabilitySlotsApiParams = {
+  userId: number;
+  after: string;
+  before: string;
```

Comment:
Before and after should be optional. 

```suggestion
  after?: string;
  before?: string;
```